airflow/providers/amazon/aws/hooks/batch_client.py

```diff
@@ -538,3 +538,35 @@ def exp(tries):
         delay = 1 + pow(tries * 0.6, 2)
         delay = min(max_interval, delay)
         return uniform(delay / 3, delay)
+
+
+class AwsBatchProtocol(BatchProtocol, Protocol):
+    """
+    This class is deprecated.
+    Please use :class:`airflow.providers.amazon.aws.hooks.batch.BatchProtocol`.
+    """
+
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "This class is deprecated. "
+            "Please use :class:`airflow.providers.amazon.aws.hooks.batch.BatchProtocol`.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        super().__init__(*args, **kwargs)
```

Comment:
Please see this discussion before approving:  https://github.com/apache/airflow/pull/20332#discussion_r770179588

Reply:
probably not needed any more...

Reply:
That is correct, based on the other one now working I'll be doing this next.   I'm not sure why that was giving me an error before, but it seems clear now. /shrug

Reply:
You were right. We need the protocol. 
see a fix we had to deploy https://github.com/apache/airflow/pull/20669